src/api_server/Cargo.toml

```diff
@@ -25,12 +26,14 @@ attestation-service = { git = "https://github.com/confidential-containers/attest
 base64.workspace = true
 cfg-if = "1.0.0"
 env_logger.workspace = true
+jsonwebtoken = {version = "8", default-features = false, optional = true }
 jwt-simple = "0.11.4"
 kbs-types = "0.2"
 lazy_static = "1.4.0"
 log.workspace = true
 prost = { version = "0.11", optional = true }
 rand = "0.8.5"
+reqwest = { version = "0.11.13", features = ["json"], optional = true }
```

Comment:
Could we just use a 0.11 `reqwest`? The `attestation-agent`already pulls that crate, so let's try to not use different crates for this one.

Reply:
Thanks your comments, I will modify.